hack/make/.binary

```diff
@@ -50,10 +50,14 @@ if [ "$(go env GOOS)/$(go env GOARCH)" != "$(go env GOHOSTOS)/$(go env GOHOSTARC
 	esac
 fi
 
-# -buildmode=pie is not supported on Windows.
-if [ "$(go env GOOS)" != "windows" ]; then
-	BUILDFLAGS+=( "-buildmode=pie" )
-fi
+# -buildmode=pie is not supported on Windows and Linux on mips.
+case "$(go env GOOS),$(go env GOARCH)" in
+	windows,*|linux,mips|linux,mipsle)
```

Comment:
Do `mips64` and `mips64le` need the same adjustment, or is `-buildmode=pie` supported there? (ie, should this be `linux/mips*`, or is this already sufficient)

Reply:
It would seem that the answer is yes: https://github.com/golang/go/blob/52cc9e3762171bd45368b3280554bf12a63f23b2/src/cmd/go/internal/work/init.go#L149 (though I was only really considering mips32 at this time)